plugins/filter/tests/tests.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testdefinition version="1.0">
+  <suite name="voicecall-filter" domain="mw">
+    <set name="unit-tests" feature="voicecall-filter">
+       <case manual="false" name="tst_filter">
+         <step>HOME=$PWD /opt/tests/voicecall/filter/tst_filter</step>
```

Comment:
In case the earlier comment got lost in the update: would you mind hard-coding the HOME to something so this doesn't surprisingly fail when executing from the home directory.

Reply:
Mmh, is it failing if actually run in the real home directory ? I'm doing this HOME=... not to damage the DConf database, when run on device by the user (me). Maybe the best way would be to hardcode the HOME directory in the test to a temporary directory... I'm going to try this.

Reply:
Yea, I often just run from the home directory like "testrunner-lite  -f /opt/tests/voicecall/filter/tests.xml -o testrun" and now it started with failure on everything.

Was ok enough if the HOME= was set to directory of the test, don't think it even needs to be a temporary one.

Reply:
Well, I'm often running the tests like `scp tst_foo nemo@192.168.2.15:~/devel/` and then `cd devel; ./tst_foo`. Having a hardcoded path to something in `/opt/` while I'm not installing the package would make the test fails.

Reply:
Yea, it wouldn't then work if you want to run the test directly without the xml part. Then again on many places we assume that the xml sets the environment right for the test.

If you get the test running without xml nice, that's nice, but maybe as first step just the HOME in .xml could be adjusted so there wouldn't be this surprise of current working directory affecting the test xml execution?